src/components/Search/SearchBar.tsx

```diff
@@ -21,21 +41,23 @@ class SearchBar extends Component {
             const child = {
                 key: `${key}-county`,
                 button: []
-            } 
+            };
             child.button.push(
+                // @ts-ignore
                 <Option
                     key={`${key}-county`}
                     value={key}>
                     {value}
                 </Option>
-            )
+            );
             children.push(child);
         }
-        this.setState({children})
+        this.setState({ children })
     }
 
-    handleCountySelect = (event) => {
+    handleCountySelect = (event: SelectValue) => {
         this.props.onCountySelect(event);
+        // @ts-ignore COUNTIES needs a type
         this.setState({countyName: COUNTIES[event]})
```

Comment:
@lxu213 `COUNTIES` needs a type, otherwise we have no idea that it's a string which is what `countyName` needs

Reply:
https://github.com/HopkinsIDD/covid-dashboard-app/pull/102/files#diff-c9f3cecdaf68c836219973ba2a781d3aR2-R4